zim/gui/pageview/__init__.py

```diff
@@ -6162,11 +6165,13 @@ def do_mark_set(self, buffer, iter, mark):
 			self.actiongroup.get_action('toggle_checkbox').set_sensitive(True)
 			self.actiongroup.get_action('xtoggle_checkbox').set_sensitive(True)
 			self.actiongroup.get_action('migrate_checkbox').set_sensitive(True)
+			self.actiongroup.get_action('transmigrate_checkbox').set_sensitive(True)
```

Comment:
This doesn't fix it not showing up in the menu, but doesn't hurt either

Reply:
See https://github.com/zim-desktop-wiki/zim-desktop-wiki/pull/1332#issuecomment-849820609 for context